packages/design-system/src/components/N8nRoute/Route.vue

```diff
@@ -1,8 +1,8 @@
 <template>
-	<router-link v-if="useRouterLink" :to="to" v-bind="$attrs">
+	<router-link v-if="useRouterLink && to" :to="to" v-bind="$attrs">
 		<slot></slot>
 	</router-link>
-	<a v-else :href="to" :target="openNewWindow ? '_blank' : '_self'" v-bind="$attrs">
+	<a v-else :href="to as string" :target="openNewWindow ? '_blank' : '_self'" v-bind="$attrs">
```

Comment:
could we avoid type coercion? maybe check if it's a string in `v-else-if`

Reply:
Updated.